events/wmt20.md

```diff
@@ -225,7 +225,7 @@ Full results of the shared tasks: [*Findings of the 2020 Conference on Machine T
 ### News translation
 
 #### → English
-The human evaluation was based on segment ratings with document context.
+The human evaluation used reference-based (monolingual) segment ratings with document context.
```

Comment:
Maybe we can turn this into a title or into two different sentences so that no info is lost? As it is presented right now, there are two categories: with and without document context.

Also, according to the [Style guide](https://machinetranslate.org/style#minimal-content), we should try to avoid parenthesis.

Reply:
Thanks for the comments, @cefoo. 
So there are basically two types of direct assessment: reference-based which is the same as monolingual and source-based which is the same as bilingual. I can suggest either deleting the `reference-based` and `source-based` phrases or adding one more sentence to explain their meanings, e.g.

> The human evaluation used monolingual segment ratings with document context. 

> The human evaluation used bilingual segment ratings with document context.

or 

> The human evaluation used monolingual segment ratings with document context. In this configuration, the human assessors should rate the system output by comparing it to the human-generated reference without considering the source input.

> The human evaluation used bilingual segment ratings with document context. In this configuration, the human assessors should rate the source input and system output without any reference translation.

Reply:
I agree with deleting that info, as I find `monolingual` and `bilingual` clearer.

Somehow, I still find it hard to understand how this introductory sentence is connected to the table.  Since these are the top systems found through different types of human evaluations, maybe we can make this relation clear by rephrasing to something like 

> Results from human evaluation using monolingual segment ratings with document context

Am I correct?

Reply:
As WMT uses only human evaluation to determine results and present them in the findings, I didn't want to use the word `results` everywhere because we already have `Results` as a section title. But I get your point, so maybe we should add an overview of the section right after the title. Something like

> ## Results
> The section presents the official results of the news translation task determined through human evaluation. Full results of the shared tasks: [*Findings of the 2020 Conference on Machine Translation (WMT20)*](https://aclanthology.org/2020.wmt-1.1.pdf)